velox/common/memory/SharedArbitrator.cpp

```diff
@@ -433,15 +433,26 @@ uint64_t SharedArbitrator::shrinkCapacity(
     uint64_t requestBytes,
     bool allowSpill,
     bool allowAbort) {
+  std::unique_lock<std::shared_mutex> exclusiveLock(
```

Comment:
Can we keep the lock the same position as we might grab the lock inside operations like ScopedArbitration? Thanks!